guides/common/modules/con_prerequisites-for-configuring-project-with-keycloak-quarkus-authentication.adoc

```diff
@@ -12,9 +32,14 @@ ifndef::orcharhino[]
 For more information about configuring a different context path, see the {RHDocsBaseURL}red_hat_build_of_keycloak/24.0/html-single/server_guide/index#reverseproxy-different-context-path-on-reverse-proxy[_{RHBK} Administration Guide_].
 endif::[]
 ====
-* If the certificates or the CA are self-signed, add them to the end-user certificate truststore.
-* A {keycloak} realm created for {Project} user accounts, for example `_{Project}_Realm_`.
-* Users imported or added to {keycloak}.
++
+If `keycloak-httpd-client-install` version 1.3 or later is installed, your {keycloak} server does not need to be initialized with the `--http-relative-path=/auth` context path.
```

Comment:
Should this also be in `ifeval::["{context}" == "keycloak-quarkus"]`?

Reply:
Good catch! It looks like I actually decided to split the quarkus and wildfly prerequisites into separate files so we don't need any `ifeval`s here anymore. (With the new steps for `keycloak-httpd-client-install`, the prerequisites got so different that it seemed easier to me to just have two different files and risk some amount of content duplication.)

TL;DR: I dropped the `ifeval`s from this file and that should be enough.